src/BackendServiceFactory.js

```diff
@@ -20,20 +20,24 @@ const objectPath = require('object-path');
 module.exports = class BackendServiceFactory {
   constructor(params) {
     this.eventParams = params;
+    const backendService = objectPath.get(this.eventParams, 'eventData.object.spec.backendService', '').toLowerCase();
+    let options = objectPath.get(this.eventParams, 'options', {});
```

Comment:
```suggestion
    const backendService = (this.eventParams?.eventData?.object?.spec?.backendService || '').toLowerCase();
    let options = this.eventParams?.options || {};
```

Reply:
most of this isn't needed with the single env var setup